packages/commonwealth/client/scripts/controllers/server/user.ts

```diff
@@ -289,13 +289,21 @@ export class UserController {
     this._setStarredCommunities(star);
   }
 
+  public isCommunityStarred(chain: string): boolean {
+    return (
+      this._starredCommunities.findIndex((c) => {
+        return c.chain === chain;
+      }) !== -1
+    );
+  }
+
```

Comment:
Starred communities are calculated during app init. So for now added this method as it seemed the best option.